bokehjs/src/lib/models/dom/template.ts

```diff
@@ -4,28 +4,38 @@ import {PlaceholderView} from "./placeholder"
 import type {Formatters} from "./placeholder"
 import type {ColumnarDataSource} from "../sources/columnar_data_source"
 import type {Index} from "core/util/templating"
-import type {ViewStorage, IterViews} from "core/build_views"
+import type {ViewStorage, IterViews, ViewOf} from "core/build_views"
 import {build_views, remove_views, traverse_views} from "core/build_views"
 import type {PlainObject} from "core/types"
 import type * as p from "core/properties"
 
 export class TemplateView extends DOMElementView {
   declare model: Template
 
-  readonly action_views: ViewStorage<Action> = new Map()
+  readonly _action_views: ViewStorage<Action> = new Map()
+  get actions(): Action[] {
+    return this.model.actions
+  }
+  get action_views(): ViewOf<Action>[] {
```

Comment:
Sure, but all models in `models.dom` are experimental, so there should be no complaints.

Reply:
Still, It would be nice to document this change _somewhere_. These small changes (even for experimental features) will compound and make upgrading harder.  

I suggest adding a comment in the code that this change has been made and in what version (3.7).

Reply:
We don't have a place for communicating such changes. For now I added a section to 3.7 release notes.